modules/drivers/snowflake/deps.edn

```diff
@@ -2,4 +2,4 @@
  ["src" "resources"]
 
  :deps
- {net.snowflake/snowflake-jdbc {:mvn/version "3.15.1"}}}
+ {net.snowflake/snowflake-jdbc {:mvn/version "3.17.0"}}}
```

Comment:
[They say 3.19 is out now](https://docs.snowflake.com/en/release-notes/clients-drivers/jdbc-2024), any reason not to bump to latest?

```suggestion
 {net.snowflake/snowflake-jdbc {:mvn/version "3.19.0"}}}
```